images/BUILD

```diff
@@ -75,6 +75,26 @@ container_layer(
     ],
 )
 
+# Layer for etcd 3.3.15, updated recommendation for k8s 1.16 and later
+container_layer(
+    name = "etcd-3-3-15-layer",
+    directory = "/opt/etcd-v3.3.15-linux-amd64/",
+    files = [
+        "//:etcd-v3.3.15-linux-amd64_etcdctl",
+        "//:etcd-v3.3.15-linux-amd64_etcd",
+    ],
+)
+
+# Layer for etcd 3.3.17, updated recommendation for k8s 1.16.3 and later
+container_layer(
```

Comment:
Do you think we could just do 3.3.17?  These are pretty expensive with our current packaging strategy, and I'd like to kick that can a little further down the road....

Reply:
that sounds good to me. updated 👍